docs/install/install-node.rst

```diff
@@ -159,11 +159,8 @@ Paste them into a root console on the target node:
 .. task:: install_flocker centos-7
    :prompt: [root@centos]#
 
-Installing ``flocker-node`` will automatically install Docker, but the ``docker`` service may not have been enabled or started.
-To enable and start Docker, run the following commands in a root console:
-
-.. task:: enable_docker centos-7
-   :prompt: [root@centos]#
+Flocker's container management features depend on Docker.
+Make sure `a recent version of Docker is installed <https://docs.docker.com/docker/installation/>` and running.
```

Comment:
Yep, 1.8 is the minimum required version if we want all the issues I claimed this PR fixes to actually be fixed.  1.8 is also what you'll get (as of today) if you follow the linked installation instructions.

I could put some words about installing 1.8 here.  I have mixed feelings about it.  Further input welcome.